Lib/test/test_asyncio/test_futures.py

```diff
@@ -32,6 +32,25 @@ def last_cb():
     pass
 
 
+class ReachableCode(Exception):
```

Comment:
Is this necessary? I think just Exception would work just fine

Reply:
With `self.assertRaises(Exception, ...)`, I don't want to catch all possible exceptions, hence the specialized one. I could use `ZeroDivisionError` though but I always feel it's weird to use this for another semantic.